backend/alert/templates/alert/push_notif.txt

```diff
@@ -0,0 +1,2 @@
+{{course}} is now open! Register on PennInTouch: http://bit.ly/pitouch. {% if auto_resubscribe %}To cancel alerts, visit{% else %}Resubscribe at{% endif %} http://penncoursealert.com#manage
+ - Thanks for using Penn Course Alert!
```

Comment:
I'm pretty sure that users can't click on individual links in mobile push notifications. Ideally, when we roll out the feature, the notification will have actions (accessed via the "...") like "go to pennintouch", "cancel alert", "resubscribe for alert", "manage alert in pennmobile".

We don't need to change it now, because I think it will remind us to add actions later, but just thought I'd give the heads up.

Reply:
Not sure how this works on the mobile side, but does the notification go straight to pennintouch when tapped?